CHANGELOG.rst

```diff
@@ -15,7 +15,7 @@ CHANGELOG
 
 **Backwards incompatible changes**
 
-* Drop support for end-of-life Django 1.11 and 2.2.
+* Drop support for end-of-life Django 1.11 and < 2.2.
```

Comment:
Only dropped support for Django 2.1 in 7fed681f8c644825af8191705c033179536f3f99, can be specific.
```suggestion
* Drop support for end-of-life Django 1.11 and 2.1.
```